recipes/jsoncpp/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.10.6" %}
+
+package:
+  name: jsoncpp
+  version: {{ version }}
+
+source:
+  fn: jsoncpp-{{ version }}.tar.gz
+  url: https://github.com/open-source-parsers/jsoncpp/archive/{{ version }}.tar.gz
+  sha256: 7c285fc40ad0c113e436a1271c4e38b5017b5c7782c306e90be9d6b2ffa90212
+
+build:
+  number: 0
+  skip: true  # [win or not py27]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - pkg-config  # [unix]
+    - python
+
+test:
+  commands:
+    # Verify headers are present.
+    - test -d "${PREFIX}/include/jsoncpp"           # [unix]
+
+    # Verify pkg-config file is present.
+    - test -f "${PREFIX}/lib/pkgconfig/jsoncpp.pc"  # [unix]
+
+    # Verify libraries are present.
+    - test -f "${PREFIX}/lib/libjsoncpp.a"          # [unix]
+    - test -f "${PREFIX}/lib/libjsoncpp.so"         # [linux]
+    - test -f "${PREFIX}/lib/libjsoncpp.dylib"      # [osx]
+
+about:
+  home: https://github.com/open-source-parsers/jsoncpp
+  license: Public Domain/MIT
```

Comment:
This is dual licensed. It aims to be Public Domain whenever possible, but in some areas it claims Public Domain is not accepted so it will be under MIT in those cases.